langserve/serialization.py

```diff
@@ -87,6 +89,8 @@ def default(obj) -> Any:
     """Default serialization for well known objects."""
     if isinstance(obj, BaseModel):
         return obj.dict()
+    if isinstance(obj, Send):
+        return {"node": obj.node, "arg": obj.arg}
     return super().default(obj)
```

Comment:
Use of super without a class context is a bug here. Can we give the default serializer more powers to allow any object to be become json serializable?

```suggestion
    try:
        return obj.__dict__()
    except:
        pass

    raise TypeError(f"Object of type {obj.__class__.__name__} is not JSON serializable")

```